README.md

```diff
@@ -11,13 +11,62 @@ This project uses deep learning computer vision to label images taken by motion-
 - [Uptake.org Autofocus Case Study](https://www.uptake.org/impact/special-projects)
 - [Machine Learning Meets Wildlife Conservation](https://www.lpzoo.org/blog/machine-learning-meets-wildlife-conservation)
 
+## Getting the App
+
+If you just want to get labels for your images, you can use the following steps to run a service that passes images through a trained model.
+
+1. Make sure [Docker](https://www.docker.com/get-started) is installed and running.
+2. Run `docker pull gsganden/autofocus` to download the app image. (Note that it takes up about 4GB of disk space.)
```

Comment:
this defaults to `gsganden/autofocus:latest`, is that what you want?

Reply:
Sure? Would you suggest a different approach?

Reply:
I'm going to go ahead and merge, but I'd welcome more input on this point. Perhaps I should be explicit about version numbers?